docs/ssl-input.asciidoc

```diff
@@ -0,0 +1,71 @@
+SSL/TLS is disabled by default. Besides enabling it, you need to provide a certificate and a corresponding
+private key as well.
+
+Following is a basic APM Server SSL config with secure communication enabled:
+
+[source,yaml]
+----
+apm-server.ssl.enabled: true
+apm-server.ssl.certificate: "/path/to/apm-server.crt"
+apm-server.ssl.key: "/path/to/apm-server.key"
+----
+
+This will make APM Server serve HTTPS requests instead of HTTP.
```

Comment:
I'd add this line to line 4 above:

> The following is a basic APM Server SSL config with secure communication enabled.
This will make APM Server serve HTTPS requests instead of HTTP.